substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1323,6 +1324,8 @@ private int completeImageBuild() {
         if (config.modulePathBuild && !finalImageClasspath.isEmpty()) {
             imageBuilderJavaArgs.add(DefaultOptionHandler.addModulesOption + "=ALL-DEFAULT");
         }
+        assert !enableNativeAccessModules.isEmpty();
+        imageBuilderJavaArgs.add("--enable-native-access=" + String.join(",", enableNativeAccessModules));
```

Comment:
@zapster For our builds this produces warnings like the following:

```
WARNING: Unknown module: com.oracle.svm.svm_enterprise specified to --enable-native-access
WARNING: Unknown module: com.oracle.graal.graal_enterprise specified to --enable-native-access
```

Could we get something like https://github.com/jerboaa/graal/commit/74a2e81efb6c71367a41ee6e864d8ac3c4d4e416 added to this PR? That would fix it for us. Thanks!

Reply:
Thanks for the suggestion. I don't want to add it to this PR though, as I want to get the update merged asap. I don't want to risk breaking things with last minute changes. Also I'm not 100% sure this is the solution we want. I first want to explore alternatives, e.g., routing all native accesses through a single module and remove all others from the native access list. But this needs more experimentation (and probably discussion).

Reply:
OK, that's fair enough. Please keep me in the loop if possible. This will fail in our CI.

Reply:
will do!

Reply:
PR is up #9654